worlds/animal_well/options.py

```diff
@@ -193,6 +243,10 @@ class AnimalWellOptions(PerGameCommonOptions):
         BubbleJumping,
         DiscHopping,
         WheelTricks,
+        BallThrowing,
+        ObscureTricks,
+        PreciseTricks,
+        TankingDamage,
         WeirdTricks,
```

Comment:
```suggestion
```
No need to put old options in option groups

Reply:
Another one lost in the recovery. Good find.